packages/auth/amplify_auth_cognito_dart/lib/src/state/state/configuration_state.dart

```diff
@@ -3,6 +3,33 @@
 
 import 'package:amplify_auth_cognito_dart/src/state/state.dart';
 import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_secure_storage_dart/amplify_secure_storage_dart.dart';
+import 'package:meta/meta.dart';
+
+/// Token for storing the [SecureStorageInterface] used to store Pinpoint Endpoint Id.
```

Comment:
I can't use the base Token class as secure storage is saved twice, once for Auth and another time for Analytics.

Reply:
This doesn't belong here - let's put in its own file

Reply:
File removed due to switch to EndpointInfoStoreManager we discussed yesterday.